keras_cv/models/backbones/vgg19/vgg19_backbone_presets.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""VGG19 model preset configurations."""
+
+backbone_presets_no_weights = {
+    "vgg19": {
+        "metadata": {
+            "description": (
+                "VGG19 model with 19 layers where the ReLU activation is "
```

Comment:
Oddly enough, the VGG19 backbone actually has 16 layers. VGG19 has 16 conv layers and 3 dense layers, and the backbone omits those dense layers.